### ISSUE TITLE
How to disable postscribe waiting for call to finish?

### ISSUE THREAD

**Issue by vindberg** [krux/postscribe#354]
Im looping through several ads calling postscribe(tagElement, adTag) on each one. It seems like postscribe is waiting for each ad script to load before it takes the next. Is it possible to disable this?

I would like it to load all ad script tags without waiting for anything.

Thanks!

**Comment by jnewman** [comment]
Pass `releaseAsync: true` in the options object

**Comment by vindberg** [comment]
Thanks mate! I tried that already, but didn't notice any difference. Will put it in production and see if it makes any changes there.

Thanks.

**Comment by tomgallagher** [comment]
hi I'm up against the same problem. Did you find a fix?

**Comment by jnewman** [comment]
@tomgallagher this sounds like you have a different problem. @vinberg wanted the scripts to insert in parallel, which is helpful sometimes, but is usually not the right thing for ad tags (if they expect to be doc.written, then they might rely on the order of execution).

Please file a second ticket with some examples and I'll try and take a look this weekend

**Comment by tomgallagher** [comment]
Hi @jnewman 

Thanks a lot for getting back. I've got to the bottom of what's happening. I've filed a new ticket #452.
 
Tom
